doc/sphinx-guides/source/admin/external-tools.rst

```diff
@@ -37,6 +37,8 @@ In the curl command below, replace the placeholder "fabulousFileTool.json" place
 
   curl -X POST -H 'Content-type: application/json' http://localhost:8080/api/admin/externalTools --upload-file fabulousFileTool.json 
 
+If you'd like to allow the external tool to be used in previews on the file page, you can define it in the manifest using the `hasPreviewMode` parameter.
```

Comment:
I'm not sure I agree. We do want an installation to control how they use these tools and preview is key. Just because a tool supports preview doesn't mean we want it on in any installation (some may prefer just one tool for preview, even if multiple are configured as external tools). 

We could consider changing the external tool manifest to be "supportsPreview" or something like that and a separate API for admins to turn preview on, if they want it (that would only work if supports preview is true.

Reply:
So when you say "a separate API for admins to turn preview on" would that be at the tool level or installation level?

Reply:
per tool, if that’s how we decided to go

Reply:
Yeah, I added this because I see the tool developer as the one responsible for providing the different possible views, but the administrator is the one that picks whether or not the tool shows in preview mode or not. No additional APIs needed right now. :) I will probably reword this, I meant to do another pass of the docs once dev was complete.

Reply:
reworded in https://github.com/IQSS/dataverse/commit/a992e0efe0ab817bc0afec5b9bf355868440e72a

Reply:
Talked about in standup, this is all set.